azavea/franklin
# Enable read from http urls

## Overview

This PR adds support for reading any json decodable type from *unauthenticated* http urls. I have no idea how / if we'll add authentication.

### Checklist

- ~New tests have been added or existing tests have been modified~

### Testing Instructions

- try to import the Sentinel-5p catalog mentioned in #323 (catalog root is: https://meeo-s5p.s3.amazonaws.com/catalog.json)
- observe that it's an http link and that you get nicer error messages
- start up the server with the `--run-migrations` flag: `AWS_PROFILE=raster-foundry AWS_REGION=us-east-1 ./scripts/server --run-migrations`
- observe that migrations were run before the server came up

Closes #259 

Closes #319
